### PR TITLE
code fencing + general review: src/en/charms+charms-config.md

### PR DESCRIPTION
**Pull request by petermatulis**

### CODE REVIEW

File: src/en/charms-config.md [hunk 1]
```diff
@@ -3,39 +3,49 @@
...
 We can then use this configuration when we deploy the service:
 
-    juju deploy --config myconfig.yaml mediawiki
+```bash
+juju deploy --config myconfig.yaml mediawiki
+```
+
+**Caution:** If the yaml configuration file cannot be read or contains some
+syntax errors or invalid options, you will receive an error message to this
+effect. However, **the service will still be deployed **.
```
Comment (evilnick) [inline src/en/charms-config.md]: hmm. You might want to check if this still renders okay

File: src/en/charms.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 The magic behind Juju is a collection of software components called charms.
-These are the encapsulated knowledge of how to properly deploy and configure
-the services you will want to deploy on your cloud. The charms available online
-in the [Charm Store](/docs/authors-charm-store.html) represent the distilled
-DevOps knowledge of experts, and they are freely available for you to use (or
-you can write your own).
+They contain all the instructions necessary for deploying and configuring
+cloud-based services.  The charms publicly available in the online [Charm
+Store](/docs/authors-charm-store.html) represent the distilled DevOps knowledge
```
Comment (evilnick) [inline src/en/charms.md]: i think this link should be just (authors-charm-store) now

File: src/en/charms.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+of experts. If you cannot find what you're looking for or if what's available
+does not suit your needs then you are free to write your own.
 
 <iframe style="margin-left: 20%;" class="youtube-player" type="text/html" width="420" height="350" src="//www.youtube.com/embed/0AT6qKyam9I"></iframe>
 
 Charms make it quick and easy to reliably and repeatedly deploy services in
-your cloud. If you haven't already tried the
-[Juju walkthrough](/docs/getting-started.html), you should give it a go and see
-how ridiculously easy it is to deploy services in minutes.
+your cloud. If you haven't already tried the [Juju
```
Comment (evilnick) [inline src/en/charms.md]: try not to break links!

**Comment by evilnick** [comment]
otherwise good!

File: src/en/charms-config.md (hunk 1 above)
Comment (petermatulis) [inline src/en/charms-config.md]: is ok

**Pull request merged**